app_feup/lib/view/Widgets/exam_card.dart

```diff
@@ -13,11 +13,11 @@ import 'package:uni/view/Widgets/schedule_row.dart';
 
 import 'generic_card.dart';
 
-Color examColor(Exam exam, context) {
+bool isHighlighted(Exam exam, context) {
   return (exam.examType.contains('''EN''')) ||
           (exam.examType.contains('''MT'''))
-      ? Theme.of(context).hintColor
-      : Theme.of(context).backgroundColor;
+      ? true
+      : false;
```

Comment:
You can simply return the expression here

Reply:
Done! Thanks!